docs/source/integrations/cvat.rst

```diff
@@ -482,7 +482,8 @@ provided:
     This argument can be a list of usernames when annotating videos as each
     video is uploaded to a separate task
 -   **job_assignees** (*None*): a list of usernames to assign jobs
--   **job_reviewers** (*None*): a list of usernames to assign job reviews
+-   **job_reviewers** (*None*): a list of usernames to assign job reviews. Only
```

Comment:
I recommend removing `job_reviewers` from the "Assigning users" example later in this file. Or maybe including the same "only available in CVAT v1" note when documenting the existence of the parameter, but not actually including it in the copy-paste code.

Reasoning is that, in general, our docs should always assume the latest version of all third-party deps.